### PR TITLE
Bump k8s-device-plugin, gpu-feature-discovery and node-feature-discovery versions

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

File: deployments/gpu-operator/Chart.yaml [hunk 1]
```diff
@@ -19,6 +19,6 @@ keywords:
 
 dependencies:
   - name: node-feature-discovery
-    version: v0.16.5
+    version: v0.16.6
```
Comment (ArangoGutierrez) [inline deployments/gpu-operator/Chart.yaml:22]: ++

**Review by ArangoGutierrez — APPROVED** [review]
NFD V0.16.6 ++

File: deployments/gpu-operator/values.yaml [hunk 2]
```diff
@@ -242,7 +242,7 @@ devicePlugin:
   enabled: true
   repository: nvcr.io/nvidia
   image: k8s-device-plugin
-  version: v0.17.0-rc.1-ubi9
+  version: v0.17.0-ubi9
```
Comment (elezar) [inline deployments/gpu-operator/values.yaml:245]: Not critical, but do we want to drop the `-ubi9` suffix?

**Review by elezar — APPROVED** [review]

File: deployments/gpu-operator/values.yaml (hunk 2 above)
Comment (tariq1890) [reply deployments/gpu-operator/values.yaml:245]: Sure, I can make a note of that. I see that we are no longer maintaining distro variants. So there will only a single tag for every release going forward?

**Pull request merged**